.github/ISSUE_TEMPLATE/ui-copy-request.yaml

```
@@ -0,0 +1,99 @@
+name: "UI copy request"
+description: Submit a UI copy-related request to the documentation team.
+title: "[UI copy]: "
+labels: "ui-copy"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Hello! Use this form to submit a UI copy related request to Elastic's documentation team. You should also directly contact the specific writer or docs team that's responsibile for the applicable part of the product, especially for time-sensitive requests. 
+        > [!WARNING]  
+        > THIS IS A PUBLIC REPO. DO NOT INCLUDE SENSITIVE INFORMATION HERE.
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Summarize your request here. What is the work and how can we help? Do we need to write new copy, or edit existing copy? Feel free to include screenshots.
+      placeholder: |
+        What: We're introducing new feature A. We need a writer to review the UI copy.
+        When: This feature will launch at the completion of project B.
+        Why: This feature will make X, Y, and Z easier for the user.
+    validations:
+      required: true
+  - type: textarea
+    id: related
+    attributes:
+      label: Related links / assets
+      description: Where can the documentation team learn more about this feature?
+      value: |
+        Please include each of the following, if applicable:
+        Figma link(s):
+        Github epic link(s):
+        Github issue link(s):
+        How to find the text in a production environment:
+        Testing environment information:
+        NOTE: To keep sensitive information such as credentials secure, do not include it in this form.
+    validations:
+      required: true
+  - type: dropdown
+    id: doc-set
+    attributes:
+      label: Which documentation set does this change impact?
+      description: ESS, serverless, or both?
+      options:
+        - ESS and serverless
+        - ESS only
+        - Serverless only
+        - Unknown / Not applicable
+      default: 0
+    validations:
+      required: true
+  - type: textarea
+    id: doc-set-differences
+    attributes:
+      label: Feature differences
+      description: If you selected both ESS and serverless above, please describe how, if at all, the feature differs in each deployment method.
+      placeholder: The feature is identical in ESS and serverless.
+    validations:
+      required: true
+  - type: textarea
+    id: software-version
+    attributes:
+      label: Software version
+      description: Is this work targeted for a particular release version? If so, which one?
+      placeholder: |
+        For example:
+        "8.19.0"; 
+        "This feature has been in production since 8.15.0"; 
+        "N/A"
+    validations:
+      required: true
```

I think we can simplify this for UI copy and just request to provide the area. Because:
- "documentation set" isn't relevant in the context of UI copy
- "doc-set-differences", whether it's serverless or not isn't relevant in the context of UI copy
- "software-version" isn't relevant as we have a Timeline field already later in the form. Design tasks don't always have a release version set.

Instead we could request to specify the product area that is concerned to make sure the issue is routed to the right folks (we can discuss the options :D):

```suggestion
  - type: dropdown
    id: product-area
    attributes:
      label: Which product area does this mainly concern?
      options:
        - Cloud UI
        - Search solution
        - Observability solution
        - Security solution
        - Kibana Analytics (Discover, Dashboards...)
        - Kibana Management
        - Something else
      default: 0
    validations:
      required: true
```